guides/release/tutorial/part-1/reusable-components.md

```diff
@@ -47,6 +47,7 @@ module.exports = function(environment) {
     APP: {
       // Here you can pass flags/options to your application instance
       // when it is created
+      RAISE_ON_DEPRECATION: true,
```

Comment:
hmm this is definitely not intended... but I don't know if it matters

Reply:
I'm going to assume this is fine, if it ends up being confusing, we can find a workaround